cortex_certifai_azure_notebooks_NY_Taxi/Tutorial.ipynb

```diff
@@ -179,35 +179,35 @@
     "subscription_id = os.getenv(\"SUBSCRIPTION_ID\", default=\"<USER-SUBSCRIPTION-ID>\")\n",
     "resource_group = os.getenv(\"RESOURCE_GROUP\", default=\"<USER-RESOURCE-GROUP>\")\n",
     "workspace_name = os.getenv(\"WORKSPACE_NAME\", default=\"<USER-WORKSPACE-NAME>\")\n",
-    "workspace_region = os.getenv(\"WORKSPACE_REGION\", default=\"<USER-WORKSPACE-REGION>\")"
+    "workspace_region = os.getenv(\"WORKSPACE_REGION\", default=\"<USER-WORKSPACE-REGION>\")  # eg. - 'useast'"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "from azureml.core import Workspace\n",
     "\n",
-    "ws = Workspace(subscription_id = subscription_id, resource_group = resource_group, workspace_name = workspace_name)\n",
-    "# write the details of the workspace to a configuration file to the notebook library\n",
-    "ws.write_config()"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 12,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from azureml.core.workspace import Workspace\n",
-    "ws = Workspace.from_config()"
+    "try:\n",
+    "    ws = Workspace.from_config()\n",
+    "except:\n",
+    "    # Assuming you have set up a fresh resource group, and workspace, in the Azure console then deploy with the\n",
+    "    # following\n",
+    "    ws = Workspace.create(name=workspace_name,\n",
+    "                   subscription_id=subscription_id,\n",
+    "                   resource_group=resource_group,\n",
+    "                   create_resource_group=False,\n",
+    "                   location=workspace_region\n",
+    "                   )\n",
+    "    # write the details of the workspace to a configuration file to the notebook library\n",
+    "    ws.write_config()"
    ]
```

Comment:
this works out nicely, i was able to create my config.json from azure portal and works straight.

Reply:
Great :-)